### PR TITLE
New pt-BR translation

### PR DESCRIPTION
**Pull request by eduardopato41**
Really liking using this module in our group so I translated it! Thanks!

### CODE REVIEW

File: lang/pt-BR.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "CN": {
+        "SETTINGS": {
+            "DEFAULT_ICON.NAME": "Ícone Personalizado",
+            "DEFAULT_ICON.HINT": "Caminho para uma imagem aqui substituirá o ícone padrão usado para o efeito de concentração.",
+
+            "ITEM_ICON.NAME": "Usar Ícone do Item",
+            "ITEM_ICON.HINT": "Se ativado, a imagem usada para efeitos de concentração será substituída pela imagem do item.",
+
+            "PREPEND.NAME": "Prepend Effect Labels",
+            "PREPEND.HINT": "If checked, concentration effects are prepended with 'Concentration' to distinguish them from other effects.",
```
Comment (krbz999) [inline lang/pt-BR.json:11]: I've had a look-over and this all looks good. I'd be willing to merge if you could translate these missing strings as well.

File: lang/pt-BR.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        "CONCENTRATION": "Concentração",
+        "SPEAKER": "Notificação de Concentração",
+        "DETAILS": "Detalhes do Item",
+        "CONCENTRATING_ON_ITEM": "Você está concentrando em {name}.",
+
+        "CARD": {
+            "GAIN.DETAILS": "{actorName} começou a se concentrar em {itemName}.",
+            "LOSS.DETAILS": "{actorName} quebrou a concentração em {itemName}.",
+
+            "PROMPT.DETAILS": "{actorName} sofreu <strong>{damage}</strong> de dano e deve fazer uma salvaguarda de {saveType} <strong>CD {dc}</strong> para manter concentração em <a name='render-item-sheet ' data-item-uuid='{itemUuid}'>{itemName}</a>.",
```
Comment (krbz999) [inline lang/pt-BR.json:48]: The extra space here in the anchor name would need to be removed for the anchor to work.
```js
<a name='render-item-sheet' ... >
```

File: lang/pt-BR.json (hunk 1 above)
Comment (eduardopato41) [reply lang/pt-BR.json:11]: I forgot haha
I skipped at first to look into the word prepend. Hold on.

**Comment by eduardopato41** [comment]
Maybe everything's working fine now? ✌️

**Comment by krbz999** [comment]
Looks good, thank you for this.

**Pull request merged**